### PR TITLE
Add BITS queue manager DBs.

### PR DESCRIPTION
**Pull request by destijl**
[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/85)

### CODE REVIEW

**Comment by destijl** [comment]
@joachimmetz

**Comment by destijl** [comment]
friendly ping

File: definitions/windows.yaml [hunk 1]
```diff
@@ -32,6 +32,17 @@ sources:
...
 urls: ['https://github.com/libyal/winreg-kb/wiki/Time-zone-keys']
 ---
+name: BITSQueueManagerDatabases
+doc: Databases that control BITS job specs and state.
+sources:
+- type: FILE
+  attributes:
+    paths:
+      - '%%environ_allusersprofile%%\Microsoft\Network\Downloader\qmgr0.dat'
+      - '%%environ_allusersprofile%%\Microsoft\Network\Downloader\qmgr1.dat'
```
Comment (joachimmetz) [inline definitions/windows.yaml]: Make this a glob. There are indications qmgr2.dat can exists as well.

**Comment by joachimmetz** [comment]
LGTM if you change the path to define "qmgr2.dat" as well. (I opt a glob)

**Pull request merged**